data/hai/hai ships.txt

```diff
@@ -1075,6 +1075,77 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 
 
 
+# Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
+ship "Emperor Beetle"
```

Comment:
You should pick a design name consistent with Hai ships. It should be an insect that is similar in shape to the ship.  The Unfettered don't have an emperor; they're a military state. His title is, "General Choot'k, Supreme Commander of the Unfettered Hai Military." See "hai reveal 5 philosophers.txt"

Reply:
The Emperor Beetle is a kind of beetle, thus bug

Reply:
If you make the big panels in the back teardrop red, the beetle will be colorful.